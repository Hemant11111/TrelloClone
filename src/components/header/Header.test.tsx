import React from 'react';
import { render, screen } from '@testing-library/react';

import HeaderComponent from "./Header";


describe('Header component', () => {

    it('should match snapshot', () => {
        const {asFragment} = render(<HeaderComponent/>);
        expect(asFragment()).toMatchSnapshot();
    })

    it('should show Trello-clone header', () => {
        render(<HeaderComponent/>)
        expect(screen.getByText('Trello-Clone')).toBeInTheDocument();
    })

    it('should show Sign out button', () => {
        render(<HeaderComponent/>)
        expect(screen.getByText('Sign out')).toBeInTheDocument();
    })
})