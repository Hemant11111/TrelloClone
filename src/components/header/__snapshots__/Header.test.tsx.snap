// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="header"
  >
    <h2>
      Trello-Clone
    </h2>
    <button>
      Sign out
    </button>
  </div>
</DocumentFragment>
`;
